Dockerfile

```diff
@@ -9,7 +9,12 @@ RUN apt-get update && apt-get install -y \
     mono-complete \
     python3 \
     python3-venv \
-    uuid-dev
+    uuid-dev \
+    wget
+
+# Use the 2023.09.13 nightly GCC 5 toolchain for RISCV64.
+RUN wget https://github.com/riscv-collab/riscv-gnu-toolchain/releases/download/2023.09.13/riscv64-elf-ubuntu-22.04-gcc-nightly-2023.09.13-nightly.tar.gz
+RUN tar -C /opt -xzf riscv64-elf-ubuntu-22.04-gcc-nightly-2023.09.13-nightly.tar.gz
```

Comment:
A few suggestions here:
1. Add `-q` to wget, otherwise its progress output is quite verbose
2. Add `-O` to set the download name so that the long tarball path doesn't have to be repeated
3. Check the hash of the download file before untaring it, just to make sure we're not downloading anything unexpected

Something like this:
```dockerfile
RUN wget -q -O gcc-riscv.tar.gz https://github.com/riscv-collab/riscv-gnu-toolchain/releases/download/2023.09.13/riscv64-elf-ubuntu-22.04-gcc-nightly-2023.09.13-nightly.tar.gz
RUN echo "c3b849d25a43d74e7eb1cf739a8a9fadba5f997c4814732650f36ae250a8ab00 gcc-riscv.tar.gz" | sha256sum --check
RUN tar -C /opt -xzf gcc-riscv.tar.gz
```

Reply:
Those suggestions sounds good.

I applied the suggestions and tested the docker build.